srcpkgs/OpenJK/template

```diff
@@ -0,0 +1,47 @@
+# Template file for 'OpenJK'
+pkgname=OpenJK
+version=0.0.20210922
+revision=1
+archs="i686 x86_64"
```

Comment:
why?

Reply:
As far as I know, they only target i686 and x86_64. I'm also sorta new to packaging and building packages in general so feel free to correct me. :)

This may be helpful but not sure: https://github.com/JACoders/OpenJK/tree/master/cmake/Toolchains

Reply:
```
index: added `OpenJK-0.0.20210922_1' (ppc64le-musl)
```

Reply:
that means musl actually builds fine, and it also means other architectures also build fine

Reply:
as a matter of fact it appears to work fine too https://0x0.st/-YOU.png